tsl/test/expected/continuous_aggs_ddl.out

```diff
@@ -1110,5 +1110,55 @@ partial_view_owner | test_role_1
 tablespace         | 
 
 \x off
+--
+-- Test drop continuous aggregate with cascade
+--
+-- Issue: #2608
+-- Case 1
+CREATE OR REPLACE FUNCTION test_int_now()
+  RETURNS INT LANGUAGE SQL STABLE AS
+$BODY$
+  SELECT 50;
+$BODY$;
+CREATE TABLE conditionsnm(time_int INT NOT NULL, device INT, value FLOAT);
+SELECT create_hypertable('conditionsnm', 'time_int', chunk_time_interval => 10);
+     create_hypertable      
+----------------------------
+ (25,public,conditionsnm,t)
+(1 row)
+
+SELECT set_integer_now_func('conditionsnm', 'test_int_now');
+ set_integer_now_func 
+----------------------
+ 
+(1 row)
+
+INSERT INTO conditionsnm
+SELECT time_val, time_val % 4, 3.14 FROM generate_series(0,100,1) AS time_val;
+CREATE MATERIALIZED VIEW conditionsnm_4
+WITH (timescaledb.continuous, timescaledb.materialized_only = TRUE)
+AS
+SELECT time_bucket(7, time_int) as bucket,
+SUM(value), COUNT(value)
+FROM conditionsnm GROUP BY bucket WITH DATA;
+NOTICE:  refreshing continuous aggregate "conditionsnm_4"
+DROP materialized view conditionsnm_4 CASCADE;
```

Comment:
When the dependent view does not exist, `CASCADE` should not be necessary. Therefore, I think this test line should not have `CASCADE` and we should still expect the drop to work.

Reply:
This is test specifically for the CASCADE, because it had the issue. It was taken from the reproduction case here: https://github.com/timescale/timescaledb/issues/2608. Usually you don't need to use cascade

Reply:
Do we test DROP without CASCADE elsewhere?

I think it makes sense to have four test cases:

Dependent view does not exist:

- `DROP cagg;` -- success
- `DROP cagg CASCADE;`  --success

Dependent view exists:

- `DROP cagg;` -- failure
- `DROP cagg CASCADE;` -- success

Reply:
Yes, but it is not the matter of this test. I believe every other test which use caggs are doing just `DROP cagg`

Reply:
I think it would be good to have these test cases in one place so that it is easy to see that all of them work as expected without having to sift through a lot of other tests.

Reply:
added more tests